config/releases.exs

```diff
@@ -7,6 +7,14 @@ secret_key_base =
     You can generate one by calling: mix phx.gen.secret
     """
 
+listen_port = String.to_integer(System.get_env("LISTEN_PORT") || "4000")
```

Comment:
Maybe :80 as default instead?

Reply:
I prefer going with port 4000. If people start it in a context where they can't open port 80 (because they need root privileges), that could be a stumbling block, and might encourage people to run it as root, which I don't want to encourage. 

Also, it is usually a good idea to run a reverse proxy web server on port 80/443 that can handle static assets and SSL, and then proxy traffic to the application.

If people do want/need to run this app on port 80, I am sure they will know enough to read the documentation and figure it out :)

Reply:
Make sense 👍